doc/user/content/sql/show-connections.md

```diff
@@ -17,6 +17,8 @@ aliases:
 Field                | Use
 ---------------------|-----
 _schema&lowbar;name_ | The schema to show connections from. If omitted, connections from all schemas are shown. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
+**EXTENDED**         | Returns all connections.
```

Comment:
Which connections are returned without this option?

Reply:
Looking at the source code for this, it also allows connections with a schema whose database id is null, which would mean system connections, which don't exist. I think we can remove documentation for this keyword, because it only adds confusions?